plugins/aws/client_fromflags.go

```diff
@@ -46,22 +47,22 @@ func newClientFromFlags(fs tbnflag.FlagSet) clientFromFlags {
 	fs.StringVar(
 		&ff.awsRegion,
 		"aws.region",
-		"",
-		usage.Required("The AWS region in which the binary is running"),
+		"us-east-1",
```

Comment:
I think I'd prefer to keep this unset and required, to force people to think about it.

Reply:
Ok, thats fair. By nature of the default chain it will also look for `AWS_REGION` in env.

Reply:
And actually, I mispoke. Becuase we point to the value. The pointer to the default `""` will persist. meaning it will have to be passed this way as well. (which is totally fair too)